OpenRA.Mods.Cnc/Traits/TDGunboat.cs

```diff
@@ -32,6 +32,7 @@ public class TDGunboatInfo : TraitInfo, IPositionableInfo, IFacingInfo, IMoveInf
 		public override object Create(ActorInitializer init) { return new TDGunboat(init, this); }
 
 		public WAngle GetInitialFacing() { return InitialFacing; }
+		public Color GetTargetLineColor() { return Color.Green; } // This is never used, since gunboats don't accept move orders.
```

Comment:
Style nit: Comments should go on their own line. I also don't think we need a comment here, but don't care really either way.